src/Devtools/Prelude.hs

```diff
@@ -1,3 +1,9 @@
-module Devtools.Prelude (module Exports) where
+module Devtools.Prelude
+  ( module Exports
+  , FilePath
+  ) where
 
-import MPrelude as Exports
+import Data.Functor as Exports (fmap)
+import MPrelude     as Exports
+
+type FilePath = String
```

Comment:
In another iteration could change to using a better type

Reply:
The `filepath` package is already a direct dependency on this project. It is also a type alias to `String`, so this line could be replaced with:

```haskell
import System.FilePath (FilePath)
```

Reply:
Good catch.However I am planning on using the path types in https://hackage.haskell.org/package/path in final PR.

Reply:
@epicallan Ahh this is nice. I like their motivation section:

https://github.com/commercialhaskell/path#motivation

Now that I think about it, I am going to purge usage of `filepath` from my own projects.